### PR TITLE
Add OpenAI API

### PR DESCRIPTION
**Pull request by T1LT**
image

### CODE REVIEW

**Review by jannyzhao — APPROVED** [review]

File: app/api/score/_utils/getAIResponse.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// openAI API setup
+import OpenAI from "openai";
+
+const openai = new OpenAI({
+  apiKey: process.env.OPENAI_KEY,
+});
+
+const getAIResponse = async (resumeText: string, jobDescription: string) => {
+  const response = await openai.completions.create({
+    model: "text-davinci-003",
```
Comment (janclarin) [inline app/api/score/_utils/getAIResponse.ts:11]: should we use the newest models? this one's marked as legacy: https://platform.openai.com/docs/models/gpt-3-5
```suggestion
    model: "gpt-3.5-turbo",
```

File: components/ScoreForm.tsx [hunk 2]
```diff
@@ -6,6 +6,8 @@ import "react-pdf/dist/esm/Page/AnnotationLayer.css";
 import "react-pdf/dist/esm/Page/TextLayer.css";
 import type { PDFDocumentProxy } from "pdfjs-dist";
 import ScoreResult from "./ScoreResult";
+// import getAIResponse from "@/utils/getAIResponse";
+// import { parsePdf } from "@/utils/parsePdf";
```
Comment (janclarin) [inline components/ScoreForm.tsx:10]: let's remove commented out code

File: package.json [hunk 3]
```diff
@@ -15,13 +15,17 @@
...
+    "@types/pdf-parse": "^1.1.1",
     "@types/react": "18.2.20",
     "@types/react-dom": "18.2.7",
     "autoprefixer": "10.4.15",
     "eslint": "8.47.0",
     "eslint-config-next": "13.4.13",
     "next": "13.4.13",
     "next-auth": "^4.23.0",
+    "node-fetch": "^3.3.2",
+    "npm": "^9.8.1",
```
Comment (janclarin) [inline package.json:27]: do we need these new packages? feel like they were added when you were trying out stuff. npm uninstall them if not needed

**Review by janclarin — APPROVED** [review]
Can you add a screenshot of the openAI outputs to the PR description?

File: package.json (hunk 3 above)
Comment (T1LT) [reply package.json:27]: I think we need node-fetch for the OpenAI API to work, getting rid of the other ones now.

**Pull request merged**